src/Resources/skeleton/authenticator/LoginFormAuthenticator.tpl.php

```diff
@@ -24,6 +24,8 @@ class <?= $class_name; ?> extends AbstractFormLoginAuthenticator<?= $password_au
 {
     use TargetPathTrait;
 
+    public const LOGIN_ROUTE = 'app_login';
```

Comment:
This could be private, correct? We're only using it from inside this class?

Reply:
I set it as public if someone want to use this const in other place. Personally, I use it one action.

Reply:
Sure :). Seems reasonable